searx/templates/simple/elements/engines_msg.html

```diff
@@ -1,27 +1,34 @@
 <div id="engines_msg">
   {% if not results and not answers %}
   <details class="sidebar-collapsable" open>
+    <summary class="title" id="engines_msg-title">{{ _('Messages from the search engines') }}</summary>
   {% else %}
   <details class="sidebar-collapsable">
+    <summary class="title" id="engines_msg-title">{{ _('Response time') }}: {{ timings[-1].total | round(1) }} {{ _('seconds') }}</summary>
   {% endif %}
-    <summary class="title" id="engines_msg-title">{{ _('Messages from the search engines') }}</summary>
-    <div class="dialog-error" role="alert">
-      {{ icon_big('warning') }}
-      <div>
-        <p>
-          <strong>{{ _('Error!') }}</strong>
-          {{ _('Engines cannot retrieve results') }}:
-        </p>
-        {%- for engine_name, error_type in unresponsive_engines -%}
-          <p>{{- engine_name }} (
-            <a href="{{ url_for('stats', engine=engine_name|e) }}"
-               title="{{ _('View error logs and submit a bug report') }}">
-               {{- error_type -}}
-            </a>
-            ){{- '' -}}
-          </p>
-        {%- endfor -%}
-      </div>
-    </div>
+    <table class="engine-stats" id="engines_msg-table">
+      {%- for engine_name, error_type in unresponsive_engines -%}
+      <tr>
+        <td class="engine-name">
+          <a href="{{ url_for('stats', engine=engine_name|e) }}"
+             title="{{ _('View error logs and submit a bug report') }}">
+             {{- engine_name -}}
+          </a>
+        </td>
+        <td class="response-error">{{- error_type -}}</td>
+      </tr>
+      {%- endfor -%}
+      {%- for timing in timings | reverse -%}
```

Comment:
Instead of `reverse` is there a way we could to `sort by desc`? This would ensure we always display in descending time, and if `request.timings` ever decided to change its ordering, it wouldn't affect this UI.